### DISCUSSION TITLE
[Feature] Navigate /browse through photos (thumbnails list) without mouse/scrollwheel; enable keyboard PgDown/PgUp

### DISCUSSION THREAD

**Discussion by dgtal1** [immich-app/immich#8879] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I'm new to immich, but it seems the web, responsive design is mainly focused on two navigation devices:
* touch
* mouse scrollwheel

Browsing photos with scrollwheel is okay, but sometimes would be great to be able to browse quicker with PgUp and PgDown keys.  Or use arrow keys instead of scrollwheel. This is currently not possible.

It's a similar request to #6202, but to browse through thumbnails with keyboard instead of browsing the slideshow. Perhaps these two can be added at once?

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
I believe #6202 encompasses what you're asking for here.

**Reply by dgtal1** [reply]
Oh, I guess you're right. It does mention the 'Photos' page. Maybe it wasn't written clear enough or lack of formatting made me miss it.
